### PR TITLE
fix: enable mouse event after closing export csv dialog

### LINKED ISSUE #815: Sometimes mouse click doesn't work after clicking exporting csv menu.
> NOTICE: only admin user and super-admin user have access to exporting csv menu.

**Describe the bug**
After clicking '...' icon in the sessions page, and click 'X' button a couple of times,
then mouse clicking event doesn't work any area of the site(Backend.AI GUI Console) 
unless the user refreshes the page explicitly.

**To Reproduce**
문제를 재현하기 위한 순서를 자세히 적어주세요:
1. Go to 'Sessions' page.
2. Click on '...' icon next to the 'OTHERS' tab.
3. 'export csv' menu pops up.
4. click 'export csv' menu.
5. 'Exrpot Session list to CSV File' dialog appears.
6. click 'X' button.
7. repeat step 5 and step 6 (2 times will be enough)
8. See the error.

**Expected behavior**
mouse clicking should be enabled after the dialog is closed.

**Client:**
- OS: macOS 10.15.6
- App version: 20.11.0
- Browser version: Firefox 82.0(64 bit)
- NodeJS version: 13.8.0

**Server:**
- OS: macOS 10.15.6
- Output of `uname -a`: Darwin Kernel Version 19.6.0: Thu Jun 18 20:49:00 PDT 2020; root:xnu-- 6153.141.1~1/RELEASE_X86_64 x86_64
- Backend.AI version: 20.09.0a2
- Python version: 3.8.3
- Installation method: used scripts/install-dev.sh

### PR DESCRIPTION
**Pull request by lizable**
This PR resolves #815

### CODE REVIEW

File: src/components/backend-ai-session-view.ts [hunk 1]
```diff
@@ -162,9 +215,204 @@ export default class BackendAiSessionView extends BackendAIPage {
...
+    let status: any = ["RUNNING", "RESTARTING", "TERMINATING",  "PENDING", "PREPARING", "PULLING", "TERMINATED", "CANCELLED", "ERROR"];
+    if (globalThis.backendaiclient.supports('detailed-session-states')) {
+      status = status.join(',');
+    }
+    const fields = ["id", "name", "image", "created_at", "terminated_at", "status", "status_info", "access_key"];
+    if (this._connectionMode === "SESSION") {
+      fields.push("user_email");
+    }
+    if (globalThis.backendaiclient.is_superadmin) {
+      fields.push("containers {container_id agent occupied_slots live_stat last_stat}");
```
Comment (inureyes) [inline src/components/backend-ai-session-view.ts:321]: @lizable  Does 20.03 support containers {} ? Please check it and add .support conditional flag for 20.03 compatibility. If it supports, it is ok.

**Review by inureyes — CHANGES_REQUESTED** [review]
Please check 20.03 compatibility issues. (Enterprise R2 includes 20.03)

File: src/components/backend-ai-session-view.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-session-view.ts:321]: I checked and found graphql error related to field value, and fixed it.
After applying few commits, I confirmed that It works well in 20.03 with containers.

**Review by inureyes — APPROVED** [review]
LGTM

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**